### PR TITLE
Add support for ListSeparator after const (closes creditkarma/thrift-parser#12)

### PR DESCRIPTION
**Pull request by phated**

### CODE REVIEW

File: tests/test.thrift [hunk 1]
```diff
@@ -14,6 +14,8 @@ const list<i32> C3 = [ 1, 2, 3 ]
 const map<i32, string> C4 = { 1: 'a', 2: 'b', 3: 'c' }
 const bool C5 = true
 const bool C6 = false
+const bool C8 = true;
```
Comment (nnance) [inline tests/test.thrift]: semicolon?

**Comment by nnance** [comment]
+1

File: tests/test.thrift (hunk 1 above)
Comment (phated) [reply tests/test.thrift]: Semi-colon and comma are valid line terminators for most definitions. Though they are completely optional.

**Comment by phated** [comment]
Closed by e1a5d2870b706b9bc4d349c8e8e84b4fa7f20292

**Pull request closed without merging**